client/my-sites/pages/page-card-info/index.jsx

```diff
@@ -66,6 +67,12 @@ function PageCardInfo( {
 						includeBasicStatus={ true }
 					/>
 				) }
+				{ isComingSoon && (
+					<span className="page-card-info__badge">
+						<Gridicon icon="house" size={ ICON_SIZE } className="page-card-info__badge-icon" />
+						<span className="page-card-info__badge-text">{ translate( 'Coming soon page' ) }</span>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **29** times:
`translate( 'Coming Soon!' )` *ES Score: 7*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/coming-soon-simplified)